packages/animations/docs/examples/dsl/BUILD.bazel

```
@@ -0,0 +1,59 @@
+load("//tools:defaults.bzl", "ng_module", "ts_library")
+load("//packages/bazel:index.bzl", "protractor_web_test_suite")
+load("@build_bazel_rules_nodejs//:defs.bzl", "rollup_bundle")
+load("@build_bazel_rules_typescript//:defs.bzl", "ts_devserver")
+
+######################
+# App
+######################
+ng_module(
+    name = "app",
+    srcs = glob(
+        ["**/*.ts"],
+        exclude = ["**/*.spec.ts"],
+    ),
+    deps = [
+        "//packages/animations",
+        "//packages/platform-browser/animations",
+    ],
+)
+
+
+######################
+# Server
+######################
+ts_devserver(
+    name = "server",
+    bootstrap = [
+        "@ngdeps//node_modules/zone.js:dist/zone.js"
+    ],
+    entry_module = "main",
```

- this doesn't boostrap the app (the entry module factory is loaded, but not evaluated)
- ergonomics issue: mistyping the entry module name doesn't result in error, instead the error is silently ignored (see https://github.com/bazelbuild/rules_typescript/issues/336)

---

check the docs - The entrymodule should be the AMD module name of the entry module such as `"_main/src/index"ts_devserver concats the following snippet after the bundle to load the application:require(["entry_module"]);`

If you look inside your bundle you'll see the module you're trying to load is `angular/packages/animations/docs/examples/dsl/main`

The right answer here is to take a label instead of a string.

---

but `_main/src/`is something that comes from the rule and not from the user code - all the apis should always be configured from the user/source code perspective and not from whatever the rule is doing under the hood.